src/CommonITILObject.php

```diff
@@ -8672,8 +8672,10 @@ public static function getKanbanColumns($ID, $column_field = null, $column_ids =
            // Core content
             $content .= "<div class='kanban-core-content'>";
             if (isset($item['_steps']) && count($item['_steps'])) {
-                $done = count(array_filter($item['_steps'], static function ($l) {
-                    return in_array($l['status'], static::getClosedStatusArray());
+                $done = count(array_filter($item['_steps'], static function ($l) use ($item) {
```

Comment:
I may be wrong, but I guess it is possible to get the status on correct item by fixing the initial query
```diff
diff --git a/src/CommonITILObject.php b/src/CommonITILObject.php
index 419bc74386..86600fad34 100644
--- a/src/CommonITILObject.php
+++ b/src/CommonITILObject.php
@@ -8586,7 +8586,7 @@ abstract class CommonITILObject extends CommonDBTM
                                 $ticket_table => [
                                     'ON'  => [
                                         $ticket_table  => 'id',
-                                        $tt_table      => 'tickets_id_1'
+                                        $tt_table      => 'tickets_id_2'
                                     ]
                                 ]
                             ],
```